lib/actions-oclif/util/available-drives.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license
+ * Copyright 2016-2020 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { flags } from '@oclif/command';
+import Command from '../../command';
+import * as cf from '../../utils/common-flags';
+import { stripIndent, getChalk, getVisuals } from '../../utils/lazy';
+
+interface FlagsDef {
+	help: void;
+}
+
+export default class UtilAvailableDrivesCmd extends Command {
+	public static description = stripIndent`
+		List available drives.
+
+		List available drives which are usable for writing an OS image to.
+		Does not list system drives.
```

Comment:
```
System drives are not listed.
```
My problem 😊   was with mixing "List" with "Does not list" (as opposed to "List + Do not list" or "Lists + Does not list").

Reply:
I think either are correct, but I thought this was a little clearer :)